### ISSUE TITLE
version compare not working as expected due to epoch removal

### ISSUE THREAD

**Issue by furlongm** [furlongm/patchman#294]
Version compare not working correctly for the following packages:

```
Installed | Available
vim-common-8.0.1763-15.el8-x86_64 | vim-common-2:8.0.1763-13.el8-x86_64
vim-enhanced-8.0.1763-15.el8-x86_64 | vim-enhanced-2:8.0.1763-13.el8-x86_64
vim-filesystem-8.0.1763-15.el8-noarch | vim-filesystem-2:8.0.1763-13.el8-noarch
```

**Comment by ThomasReithGer** [comment]
Hello Marcus,
confirmed, same problem here for centos7 packages using 1.2.27 

bind-export-libs-9.11.4-26.P2.el7_9.4-x86_64 | bind-export-libs-32:9.11.4-26.P2.el7_9.4-x86_64
...
wpa_supplicant-2.6-12.el7-x86_64 | wpa_supplicant-1:2.6-12.el7-x86_64

![image](https://user-images.githubusercontent.com/63041231/110280764-ace22800-7fdb-11eb-8a3f-2d9711dfff07.png)

Thomas

**Comment by furlongm** [comment]
Is the server running on CentOS or Ubuntu/Debian?

**Comment by ThomasReithGer** [comment]
server runs on Ubuntu 18.04

**Comment by furlongm** [comment]
This should be fixed with 98b5fca37a7b111ac095fd356932aafe99cee1ea

Let me know if 1.2.29 works for you.

**Comment by ThomasReithGer** [comment]
No luck, Ubuntu clients work, CentOS clients not (tested with 1.2.30 client and server; patchman -a completed)
![image](https://user-images.githubusercontent.com/63041231/110742303-69372a80-8236-11eb-84db-b08d7ebfd534.png)

**Comment by furlongm** [comment]
@ThomasReithGer the old packages are not being cleaned up, as they are in use by existing package updates or errata. I would suggest using the django admin interface to delete all errata, then running `patchman -d`, then `patchman -e`. If that does not work, delete all errata _and_ all package updates, then clean database and refresh errata.

**Comment by ThomasReithGer** [comment]
Hello Marcus,
after rebuilding the database from scratch, everythings works 👍

The django admin errata removal ran into timeout errors. 
Perhaps, it would be better to avoid displaying thousands of errata rows for 
delete confirmation, which broke the browser ;-)

Good Job

Thomas

**Comment by furlongm** [comment]
This is fixed in a different way by 52817ba34d4e38a3f6ef69dadc4a7bc0456314ea
